requirements.txt

```diff
@@ -3,4 +3,5 @@ lxml==5.1.0
 numpy==1.24.4
 opencv-python==4.9.0.80
 pymavlink==2.4.41

```

Comment:
Out of curiosity, do we need to have a particular version for these libraries?

Reply:
just incase they have conflicts